rpm/mlsdb-data.spec

```diff
@@ -63,69 +63,70 @@ cp Eastern_Europe/* %{buildroot}%{_datadir}/geoclue-provider-mlsdb/data
 mkdir -p %{buildroot}%{_datadir}/translations/source
 cp mlsdb-data.ts %{buildroot}%{_datadir}/translations/source
 
-%files australia
-%defattr(-,root,root,-)
-%{_datadir}/geoclue-provider-mlsdb/data/505.dat
-
-%files india
+%files eastern-europe
```

Comment:
Is there some reason for shuffling files blocks around? That combined with switch from sorted .dat lists to unsorted ones makes diff rather noisy. Does not really matter as the files are still there in expected places (knock knock) but reaching that conclusion was rather cumbersome.

Reply:
This is autogenerated by the script.